artifactory/commands/transferfiles/errorshandler.go

```diff
@@ -236,6 +237,14 @@ func (mng *TransferErrorsMng) writeRetryableErrorContent(e ExtendedFileUploadSta
 	return nil
 }
 
+func logWritingArtifact(e ExtendedFileUploadStatusResponse, errorsFilePath string) {
+	msg := fmt.Sprintf("Writing artifact '%s' to errors file '%s'.", path.Join(e.Repo, e.Path, e.Name), errorsFilePath)
+	if e.Reason != "" {
```

Comment:
I'm not 100% sure whether this is necessary, but maybe it's a good protection -
```
if e.Reason != "" {
-->
if e != nil && e.Reason != "" {
```

Reply:
not a pointer, so cannot be nil